llvm/test/Transforms/AggressiveInstCombine/strncmp-1.ll

```diff
@@ -0,0 +1,216 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -S -passes=aggressive-instcombine -strncmp-inline-threshold=3 < %s | FileCheck %s
```

Comment:
```suggestion
; RUN: opt -S -passes=aggressive-instcombine < %s | FileCheck %s
```
I'd omit the flag here to test default behavior.

Reply:
done.